net/tstun/wrap.go

```diff
@@ -154,12 +155,19 @@ type Wrapper struct {
 	disableTSMPRejected bool
 }
 
-// tunReadResult is the result of a TUN read: Some data and an error.
-// The byte slice is not interpreted in the usual way for a Read method.
+// tunReadResult is the result of a TUN read, or an injected result pretending to be a tun read.
```

Comment:
TUN vs tun case within a few words of each other. It's not an acronym I believe but it seems we say TUN in English text.

Reply:
(as a variable, just follow normal Go https://github.com/golang/go/wiki/CodeReviewComments#initialisms rules)

Reply:
"tun" is just short for "tunnel", not an acronym. I think we inherited the "TUN" misspelling from WireGuard code, but the correct go format is "Tun".

Reply:
I made it slightly more self-consistent, with the type name always lowercased, and docstrings always uppercased.